### PR TITLE
Fix master branch

### PR DESCRIPTION
**Pull request by NazarPo**
- Introduce CircleCI config
- Remove offset data from `ntpq` to fix unit tests
- Add the logic to check if `distro_major_version` is a digit in setup.py to run tests in virtual envs

### CODE REVIEW

File: src/collectors/ntpd/test/fixtures/ntpq [hunk 1]
```diff
@@ -1,4 +1,4 @@
-     remote           refid      st t when poll reach   delay   offset  jitter
-==============================================================================
-*149.20.68.17    70.84.160.232    2 u   39 1024  377    0.127   -0.005   0.026
-+67.18.187.111   129.7.1.66       2 u  721 1024  377    0.512   -0.225   0.436
+     remote           refid      st t when poll reach   delay  jitter
+=====================================================================
+*149.20.68.17    70.84.160.232    2 u   39 1024  377    0.127   0.026
++67.18.187.111   129.7.1.66       2 u  721 1024  377    0.512   0.436
```
Comment (vladislavkoz) [inline src/collectors/ntpd/test/fixtures/ntpq]: Add new line

**Comment by vladislavkoz** [comment]
Is this critical?
image
[Task added to Asana](https://app.asana.com/0/1201106364924220/1201863952022509)

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    steps:
+      - checkout
+      - run: apt-get update
+      - run: apt-get install -y libpq-dev lm-sensors
+      - run: pip install -r .travis.requirements.txt
+      - run: pip install coverage
+      - run: make install
+      - run: coverage run --source=diamond test.py
+      - store_artifacts:
+          path: /tmp/artifacts
```
Comment (vladislavkoz) [inline .circleci/config.yml]: This step is probably redundant
image

**Comment by NazarPo** [comment]
>  Is this critical?
image

Created the ticket on Asana: https://app.asana.com/0/1201106364924220/1201863952022509

**Pull request closed without merging**